saleor/graphql/shipping/mutations/shippings.py

```diff
@@ -40,6 +47,20 @@ class ShippingPriceInput(graphene.InputObjectType):
     shipping_zone = graphene.ID(
         description="Shipping zone this method belongs to.", name="shippingZone"
     )
+    add_postal_code_rules = graphene.List(
+        graphene.NonNull(ShippingPostalCodeRulesCreateInputRange),
+        required=True,
+        description="Postal code rules to add.",
+    )
+    delete_postal_code_rules = graphene.List(
+        graphene.NonNull(graphene.ID),
+        required=True,
+        description="Postal code rules to delete.",
+    )
+    inclusion_type = PostalCodeRuleInclusionTypeEnum(
+        required=True,
+        description="Inclusion type for currently assigned postal code rules.",
+    )
```

Comment:
I initially suggested these fields to be required in one of previous my comments, but now I think they should be optional since the postal code rules are also optional. We should be able to create a shipping zone without passing any of those fields.